VM/plot_vm.py

```diff
@@ -214,12 +258,22 @@ def load_args(logger: Logger = qclogging.get_basic_logger()):
     )
 
     args = parser.parse_args()
-    args.vm_params_path = Path(args.vm_params_path).resolve()
+    args.vm_params_path = args.vm_params_path.resolve()
+    args.rel_path = args.rel_path.resolve()
```

Comment:
If the rel_path is now mandatory fro the script then we should remove the logic checking if the rel_path is None? Or actually just let the rel_path be optional? (Which might make more sense)

Reply:
Good point. rel_path is mandatory, and will skip the check.